Demo/build.gradle

```diff
@@ -67,6 +67,13 @@ dependencies {
     implementation deps.navigationUI
     implementation deps.preference
 
+    // Networking
+    implementation deps.retrofit
+    implementation deps.gson
+
+    // Dependency Injection
+    implementation deps.koin
```

Comment:
Nitpick - not sure we need comments here (since we don't have them for any other dependencies and we comment what they are for in the other file)